browser_history/browsers.py

```diff
@@ -121,3 +121,27 @@ class Edge(Browser):
 
     history_file = Chrome.history_file
     history_SQL = Chrome.history_SQL
+
+class Opera(Browser):
+    """Opera Browser
+
+    Supported platforms (TODO: Mac OS support)
+
+    * Linux, Windows
+
+    Profile support: No
+    """
+    name = "Opera"
+
+    linux_path = '.config/opera'
+    windows_path = 'AppData/Roaming/Opera Software/Opera Stable'
+    
+    profile_support = False
```

Comment:
Are you sure about this? (I'm asking because Opera is based on Chromium and Chromium/Chrome have support for profiles)

Reply:
This, not so sure, I couldn't figure out properly

Reply:
So, when I tried using the chrome profile for this too, it doesn't seem to work, and I can't figure out the profiles in the directory

Reply:
Oh okay. Opera does not seem to have enabled profiles